packages/vue-components/src/panels/MinimalPanel.vue

```diff
@@ -183,13 +183,8 @@ export default {
     margin: 0;
   }
 
-  .card-title-transparent {
-    opacity: 0.5;
-    transition: opacity 0.5s;
-  }
-
-  .card:hover .card-title-transparent {
-    opacity: 1;
+  .card-title {
+    color: black;
```

Comment:
Thanks! 🙂 
Just one nit:

hmm is this still needed? (also for `.morph-title` above)

Reply:
Oh, I explicitly put black color just as a safety if there's conflicting styles from any stylings from the panel's parent or something. Actually, I don't really know how likely that is to happen though, what do you think? If it's not likely, I can remove it.

Reply:
ahh I see, I was thinking of letting these inherit the color (default behaviour) actually since it might be more convenient / expected for the author.

We should be good here since you've ensured the critical one is `color: white` under `morph-title:hover`. (since the contrast is essential against a dark background here)

Sound good?

Reply:
Yeah, looking at it, it made more sense to inherit as a default. Okay then, will remove the explicit black color. 👍